Documentation/security/policy/language.rst

```diff
@@ -329,6 +329,10 @@ kube-apiserver
     The kube-apiserver entity represents the kube-apiserver in a Kubernetes
     cluster. This entity represents both deployments of the kube-apiserver:
     within the cluster and outside of the cluster.
+ingress
+    The ingress entity represents the Cilium Envoy instance which handles ingress
+    L7 traffic. Be aware that this also applies for pod-to-pod traffic within
+    the same cluster when using ingress endpoints (hairpinning).
```

Comment:
Deep into the nitty weeds: s/which/that, some small clarification, italics when introducing a new term
```suggestion
    The ingress entity represents the Cilium Envoy instance that handles ingress
    L7 traffic. Be aware that this also applies for pod-to-pod traffic within
    the same cluster when using ingress endpoints (also known as *hairpinning*).
```

Reply:
Handy guidance for ["which" vs. "that"](https://prowritingaid.com/art/438/-Which--or--That-%3A-Know-When-to-Use-Each.aspx#:~:text=That%3A%20Quiz-,The%20Trick%20for%20Remembering%20Which%20vs.,clause%2C%20use%20%E2%80%9Cthat.%E2%80%9D):

> If your sentence has a clause but does not need it, use “which”; if the sentence does need the clause, use “that.”